packages/emitter-framework/src/python/components/class-declaration/class-declaration.test.tsx

```diff
@@ -0,0 +1,855 @@
+import { getOutput } from "#python/test-utils.jsx";
+import { Tester } from "#test/test-host.js";
+import { List } from "@alloy-js/core";
+import * as py from "@alloy-js/python";
+import { t } from "@typespec/compiler/testing";
+import { describe, expect, it } from "vitest";
+import { ClassDeclaration } from "../../../../src/python/components/class-declaration/class-declaration.js";
+import { ClassMethod } from "../../../../src/python/components/class-declaration/class-method.js";
+import { EnumDeclaration } from "../../../../src/python/components/enum-declaration/enum-declaration.js";
+
+describe("Python Class from model", () => {
+  it("creates a class", async () => {
+    const { program, Widget } = await Tester.compile(t.code`
+
+    model ${t.model("Widget")} {
+      id: string;
+      weight: int32;
+      aliases: string[];
+      isActive: boolean;
+      color: "blue" | "red";
+      promotionalPrice: float64;
+      description?: string = "This is a widget";
+      createdAt: int64 = 1717334400;
+      tags: string[] = #["tag1", "tag2"];
+      isDeleted: boolean = false;
+      alternativeColor: "green" | "yellow" = "green";
+      price: float64 = 100.0;
+    }
+    `);
+
+    expect(getOutput(program, [<ClassDeclaration type={Widget} />])).toRenderTo(
+      `
+          from dataclasses import dataclass
+          from typing import Literal
+
+          @dataclass
+          class Widget:
+            id: str
```

Comment:
Why do we have two-space indentation? Python requires [four spaces](https://peps.python.org/pep-0008/#indentation).